docs/reference/setup/sysconfig/heap_size.asciidoc

```diff
@@ -61,7 +61,7 @@ in the jvm.options file and setting these values via `ES_JAVA_OPTS`:
 [source,sh]
 ------------------
 ES_JAVA_OPTS="-Xms2g -Xmx2g" ./bin/elasticsearch <1>
-ES_JAVA_OPTS="-Xmx4000mb -Xmx4000mb" ./bin/elasticsearch <2>
+ES_JAVA_OPTS="-Xms4000mb -Xmx4000mb" ./bin/elasticsearch <2>
```

Comment:
This should actually be:

    ES_JAVA_OPTS="-Xms4000m -Xmx4000m" ./bin/elasticsearch <2>

The JVM doesn't understand `mb`